core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -93,7 +102,7 @@
   private static final int DEFAULT_SERVER_PROXY_PORT = 3307;
   private static final int RSA_KEY_SIZE = 2048;
 
-  private static SslSocketFactory sslSocketFactory;
+  private static CoreSocketFactory sslSocketFactory;
```

Comment:
Can you rename the field to be consistent?

Reply:
Done.